### PR TITLE
Sync trove classifiers with tox.ini

### PR DESCRIPTION
**Pull request by ulgens**

### CODE REVIEW

**Comment by ulgens** [comment]
@erikwrede @nikolaik 👋🏻 🌷

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
     "coveralls",
     "mock",
     "pytz",
-    "django-filter<2;python_version<'3'",
     "django-filter>=2;python_version>='3'",
```
Comment (nikolaik) [inline setup.py:22]: can this be unconditional now, since we are python 3 only?

**Review by nikolaik — COMMENTED** [review]
Awesome!

Should we also go ahead with some of the changes in #1337 ?

**Comment by ulgens** [comment]
> Should we also go ahead with some of the changes in https://github.com/graphql-python/graphene-django/pull/1337 ?

This PR for the legacy v2 branch. We can handle similar changes for main in a different one.

**Comment by nikolaik** [comment]
> > Should we also go ahead with some of the changes in #1337 ?
> 
> This PR for the legacy v2 branch. We can handle similar changes for main in a different one.

Ah, I see, then this makes sense

File: setup.py (hunk 1 above)
Comment (ulgens) [reply setup.py]: Done.

**Review by erikwrede — APPROVED** [review]

**Pull request merged**